File: Sendo/app/src/main/java/com/example/sendo/MainActivity.java
package com.example.sendo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity {
    ImageView img;
    Button btn;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        img = findViewById(R.id.imageView4);
        btn = findViewById(R.id.Button);

        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                img.setImageResource(R.mipmap.sendo);
            }
        });
    }
}

File: ByeVirus2/app/src/main/java/com/example/byevirus/MainActivity.kt
package com.example.byevirus

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageView


class MainActivity : AppCompatActivity() {
    //buat mastiin key tidak berbeda atau tetep sama, seperti static variabel
   companion object{
       const val extra = "Extra"
   }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val arrow_click_1 = findViewById(R.id.Image_arrow) as ImageView

        arrow_click_1.setOnClickListener{
            openSecondPage()
        }

    }
    private fun openSecondPage(){
        val intent = Intent(this, SecondPage::class.java).apply {
            putExtra(extra, "This is from main activity")
        }
        startActivity(intent)
    }
}